__tests__/comments.test.js

```diff
@@ -50,7 +50,81 @@ describe("GET /api/reviews/:review_id/comments", () => {
       .expect(400);
 
     expect(error).toEqual({
-      msg: "sometext is not a valid review id",
+      msg: "sometext is not a valid review_id",
+    });
+  });
+});
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("status: 201, adds the comment to the db and responds with the added comment", async () => {
+    const testStart = new Date();
+    const { body } = await request(app)
```

Comment:
you're playing with fire by using `Date` in your tests - the output may be different depending where your code is running! (who knows where the server hosting your code will be!)

I appreciate your approach and i understand that you're comparing two timestamps created one-after-the-other (hopefully 🤞) but comparing a datetime from javascript to a datetime from psql is a recipe for inconsistant tests

If you were running your tests in a pipeline (see CI-CD lecture from today) then you'd be more likely to run into trouble

Reply:
That wouldn't have occurred to me but makes perfect sense. Will edit...